packages/react-theme-provider/package.json

```diff
@@ -41,7 +41,7 @@
     "react-test-renderer": "^16.3.0"
   },
   "dependencies": {
-    "@fluentui/react-shared-contexts": "^9.0.0-alpha.4",
+    "@fluentui/react-shared-contexts": "^9.0.0-alpha.3",
```

Comment:
This should be alpha.4 throughout

Reply:
`syncpack:fix ` is downgrading it to version `.3` @ecraig12345

Reply:
We should probably file an issue about that, either requesting to always use the newer version, or to allow selecting a strategy. I'm not sure what strategy it's using now, but it's often wrong like this.